docs/config.json

```diff
@@ -1455,6 +1455,11 @@
       "destination": "/admin-guides/access-controls/access-lists/guide/",
       "permanent": true
     },
+    {
+      "source": "/access-controls/access-lists/nested-access-lists/",
+      "destination": "/admin-guides/access-controls/access-lists/nested-access-lists/",
+      "permanent": true
+    },
```

Comment:
```suggestion
```

No need for a redirect since this is a new page. Since we generate the sidebar from the docs directory, we shouldn't need to edit `config.json` at all.

Reply:
Gotcha, wasn't sure if redirects should exist for all pages or just existing ones that prev had different URLs.